pc_filter:
  passthrough:
    x_min: -10
    x_max: 0.2
    y_min: -10
    y_max: 10
    z_min: 0.1
    z_max: 0.8
table_height: 0.6
